source/Command.cpp

```diff
@@ -61,12 +61,13 @@ const Command Command::CLOAK(1uL << 17, "Toggle cloaking device");
 const Command Command::MAP(1uL << 18, "View star map");
 const Command Command::INFO(1uL << 19, "View player info");
 const Command Command::FULLSCREEN(1uL << 20, "Toggle fullscreen");
-const Command Command::FIGHT(1uL << 21, "Fleet: Fight my target");
-const Command Command::GATHER(1uL << 22, "Fleet: Gather around me");
-const Command Command::HOLD(1uL << 23, "Fleet: Hold position");
-const Command Command::AMMO(1uL << 24, "Fleet: Toggle ammo usage");
-const Command Command::WAIT(1uL << 25, "");
-const Command Command::STOP(1ul << 26, "");
+const Command Command::FASTFORWARD(1uL << 21, "Toggle fast-forward");
```

Comment:
Any reason you went for 21 instead of 27? Going for 27 would minimize the diff

Reply:
From what I saw (didn't look too carefully, so I might have overlooked something), the constants have the same order in the command class, in the preferences panel, and here. So, I decided to keep the order, and then kept the numbers consecutive.